src/Util.ts

```diff
@@ -420,6 +420,25 @@ class Util {
             info
         };
 
+        // Get music info if there are any
+        let musicInfo: MusicInfo[] | void;
+
+        try {
+            const jsonData = html.split('{"horizontalCardListRenderer":')[1].split(',{"reelShelfRenderer"')[0]
+
+            musicInfo = JSON.parse('{"horizontalCardListRenderer":' + jsonData).horizontalCardListRenderer.cards.map((val: any) => {
+                return {
+                    title: val.videoAttributeViewModel.title,
+                    cover: val.videoAttributeViewModel.image.sources[0].url,
+                    artist: val.videoAttributeViewModel.subtitle,
+                    album: val.videoAttributeViewModel.secondarySubtitle.content,
+                    raw: val.videoAttributeViewModel
```

Comment:
I don't think we need to expose raw data here

Reply:
Ok, I'll change it!